### PR TITLE
#161 Adjust cumulative line weight

### PR DESCRIPTION
**Pull request by LIYUHAN22**
#161 Adjust cumulative line weight

Hardcode the cumulative line weight to be same as the example.

### CODE REVIEW

**Comment by LIYUHAN22** [comment]
I forgot to put the screenshot
截屏2022-12-09 下午9 35 49

File: src/cumulativeLine.ts [hunk 1]
```diff
@@ -52,5 +58,5 @@ export function renderCumulativeLine(pareto: Pareto) {
         .attr("d", line)
         .style("fill", "none")
         .style("stroke", "#3050EF")
-        .style("stroke-width", "2");
+        .style("stroke-width", 3);
```
Comment (MajaLinder) [inline src/cumulativeLine.ts:61]: Just a small thing,  I think we should avoid magic numbers and store this in a variable instead. Perhaps you can fix that, but I approve it now.

**Review by MajaLinder — APPROVED** [review]

File: src/cumulativeLine.ts (hunk 1 above)
Comment (LIYUHAN22) [reply src/cumulativeLine.ts]: I have change it now

Comment (MajaLinder) [reply src/cumulativeLine.ts]: One minor comment, I would put the variable at the top together with the other variables, or even in the resources file

Comment (LIYUHAN22) [reply src/cumulativeLine.ts]: It's on the top now

**Pull request merged**